.dev/config.toml

```diff
@@ -1,5 +1,8 @@
 httpPort = 4001
 
+[client]
+url = "http://localhost:4001"
```

Comment:
This is not related, but it will prevent `dexo` from trying to start a daemon unless running `dexo daemon` explicitly.